sources/packages/web/src/services/BypassedRestrictionService.ts

```diff
@@ -0,0 +1,27 @@
+import ApiClient from "@/services/http/ApiClient";
+import { BypassedRestrictionSummaryAPIOutDTO } from "@/services/http/dto";
+
+/**
+ * Client service layer for Bypassed Restrictions.
+ */
+export class BypassedRestrictionService {
```

Comment:
We mostly follow the same class names what is being used in API. 

in this case `ApplicationRestrictionBypassService`

and likewise the file name `ApplicationRestrictionBypassService.ts`

Reply:
Updated.